src/polls/PollForm.vue

```diff
@@ -84,22 +124,22 @@
             'p-invalid': v$.pollData.acceptanceCriteria.$error,
           }"
           class="radio-button"
-        ></RadioButton>
-        <label for="item.name">{{ item }}</label>
+        />
+        <label :for="key"> {{ item }} </label>
```

Comment:
Do you need the space after the {{ item }}?

Reply:
definitely not )
thank you for noticing!